pkgs/development/libraries/quarto/default.nix

```diff
@@ -19,10 +19,10 @@
 
 stdenv.mkDerivation (final: {
   pname = "quarto";
-  version = "1.4.557";
+  version = "1.5.52";
```

Comment:
there's a 1.5.53 already, can you update to that version insetad?

Reply:
Yes, will do!